src/sas/sasgui/perspectives/invariant/invariant_panel.py

```diff
@@ -782,7 +783,9 @@ def on_preview(self, event=None):
         report_text_str = self.state.__str__()
         report_img = self.state.image
         report_list = [report_html_str, report_text_str, report_img]
```

Comment:
May want to make sure that ReportImageHandler has been initialized, either using:
```
ReportImageHandler.check_for_empty_instance()
imgRAM = ReportImageHandler.instance.img_holder
```
or with module as singleton as above:
```
imgRAM = report_image_handler.get_handler().img_holder
```

You probably don't need to since the code is only called after a report is generated(?), but if it is easy then code defensively.